CHANGELOG.md

```diff
@@ -7,24 +7,41 @@ be found.
 
 ## 17.03.0-ce (2017-03-01)
 
+**IMPORTANT**: Starting with this release, Docker is on a monthly release cycle and uses a
+new YY.MM versioning scheme to reflect this. In addition to a monthly release, more stable
```

Comment:
Maybe this should say: "Two channels are available: monthly and quarterly. Any given monthly release will only receive security and bugfixes until the next monthly release is available. Quarterly releases receive security and bugfixes for 4 months after initial release."